_shared/hooks/post_gen_project.py

```diff
@@ -26,6 +26,16 @@ def remove_conditional_files():
         "conf/alembic.ini",
         "{{ cookiecutter.package_name }}/migrations/env.py",
         "{{ cookiecutter.package_name }}/migrations/script.py.mako",
+        "{{ cookiecutter.package_name }}/db.py",
+        "{{ cookiecutter.package_name }}/models/__init__.py",
+        "tests/factories/__init__.py",
+        "tests/factories/factoryboy_sqlalchemy_session.py",
+    ])
+    {% endif %}
+
+    {% if not (cookiecutter.get("postgres") == "yes" or cookiecutter._directory == "pyramid-app") %}
+    paths_to_remove.extend([
+        "tests/conftest.py",
```

Comment:
If the project isn't a Pyramid app and doesn't use Postgres then there's nothing to go in the cookiecutter-generated `tests/conftest.py`, it would be an empty file, so remove it